### DISCUSSION TITLE
Link User Manaul PDF to Asset Model

### DISCUSSION THREAD

**Discussion by markr-metra-autosound-com** [snipe/snipe-it#9010] [Ideas]
Would it be possible to add a field at the Asset Model level for a PDF upload of that Asset Model's user manual?

This would be super useful!

**Comment by trallnag** [comment]
Was also looking for this. The following seems to indicate that this feature already exists. But I cannot find it.

> That field is not intended for PDFs, just for images of the asset. You can add other files once the asset has been created.

https://github.com/snipe/snipe-it/issues/1962#issuecomment-211825077

**Comment by trallnag** [comment]
Ah, @markr-metra-autosound-com, I found how it works. You first create the asset. Then you go to the list of all your assets and click either on the Asset Name or the Asset Tag. Then you will see a new screen and on a menu you will be able to click on Files.

**Comment by markr-metra-autosound-com** [comment]
Hey @trallnag That could work, however the manual would be related to the Asset Model not the individual asset.

For instance, we have 33 Dell Optiplex 3040's and all 33 use the same manual. I want to be able to upload the manual 1 time at the Asset Model level, then that pulls in to each created Asset. Not upload the same PDF 33 times.

If that's what we gotta do, that's what we gotta do, but that breaks normalization, in database terms. (and takes up alot of unnecessary disk space) 

If I had my druthers, I'd make the Asset Model accept a manual, then leave the Asset level upload there for something like a repair receipt of that individual item.

**Comment by markr-metra-autosound-com** [comment]
not to toot my own horn, but albeit 4 upvotes, this is still the most upvoted "idea" users want....js....

**Comment by markr-metra-autosound-com** [comment]
Not really sure when this was implemented, I just stumbled upon it from the View Asset Model screen (NOT the edit asset model screen)

This is AWESOME!

 THANKS SNIPE + TEAM!!!
